specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/machineLearningServices.json

```diff
@@ -4996,6 +5605,461 @@
         }
       }
     },
+    //////////////////////////////////////////////////////
+    // Inference Endpoint
+    //////////////////////////////////////////////////////
+    
+    // TODO: add this to the MANIFEST to enable MSI creation 
+    // TODO: will need to work with workspaceRP to make sure deletion is reliable
+    // 
+    // "identityManagement": {
+    //   "type": "SystemAssigned, UserAssigned"
+    // }
+    "InferenceEndpointResource": {
+      "type": "object",
+      "description": "Machine Learning Inference Endpoint object wrapped into ARM resource envelope.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "properties": {
+              "description": "Inference Deployment properties",
+              "$ref": "#/definitions/InferenceEndpointResponse"
+            }
+          }
+        }
+      ]
+    },
+    "InferenceEndpointResponse": {
+      "description": "The response for an Inference Endpoint.",
+      "type": "object",
+      "properties":
+      {
+        "type": "object",
+        "properties": {
+          "name": {
+            "description": "The name of the Inference Endpoint.",
+            "type": "string"
+          },
+          "description": {
+            "description": "The service description.",
+            "type": "string"
+          },
+          "kvTags": {
```

Comment:
is this needed since we are now an ARM resource - are these ARM tags?

Reply:
these are tags specific to InferenceEndpoint. There may be tags that ARM doesn't need to know that InferenceEndpoint services do.